### ISSUE TITLE
Support for isUnique

### ISSUE THREAD

**Issue by JDziurlaj** [SteKoe/ocl.js#30]
I have several status attributes in a class, each one being an array of an enumeration type. This is to handle when an attribute can be composed of multiple statuses. However, the attribute shouldn't contain multiple values of the same enumeration literal, i.e. its values should be unique. It seems OCL has a function for this, called `isUnique`

example constraint:

```ocl
self.RejectionStatusType->isUnique(self)
```

I imagine for this to work, all `OclExpression`s must have a comparable value.

**Comment by SteKoe** [comment]
Hi John,

please check if the following changes on the feature branch would help you.
https://github.com/SteKoe/ocl.js/blob/feat/isUnique/test/expressions/collection/isUnique.spec.ts

Otherwise it would be helpful if you could provide the structure of `RejectionStatusType`. Is it a plain array of strings or an object having key/values that match like this: 

```
const RejectionStatusType = {
    A: 'A',
    B: 'B',
    C: 'C'
}
```

Regards,
Stephan

**Comment by JDziurlaj** [comment]
Hi Stephan, 

In my case the array is simply of string literals, such as `["val1","val2","dup","val3","dup"]`

Thanks for looking into this!
John.

**Comment by SteKoe** [comment]
In case of a simple array as you have specified, the branch should already work for you! Please give it a try by installing the branch via npm directly - if possible.

**Comment by JDziurlaj** [comment]
I can't install the branch from npm, but I pulled the branch from GitHub and ran it locally. LGTM!

**Comment by SteKoe** [comment]
Great, I have just published v1.2.0 to npm containing this function!
